config/core/configmaps/data-residency.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-data-residency
+  namespace: cloud-run-events
+data:
+  _example: |
```

Comment:
I'm fine with creating such a placeholder since it does nothing. But I'm looking for more details on how this will be used.

1. How this configmap will be read? Volume mount? Watch?
2. Do you allow finer grained control? Like per broker? How it would impact the configmap structure?

Reply:
I will add the actual code for this configuration to this PR too. For your question:

1. This is still being discussed, but watch seems better since our system already uses watch for other config maps, also, it provides easier interface for reacting to the change.

2. Personally I think even if we allow per broker configuration, this will still serve as a default and same configuration will be added to per broker and get reconciled by the broker controller instead.